raiden-ts/src/actions.ts

```diff
@@ -57,3 +57,36 @@ export const RaidenEvents = [
 ];
 /* Tagged union of RaidenEvents actions */
 export type RaidenEvent = ActionType<typeof RaidenEvents>;
+
+/**
+ * Set of [serializable] actions which are first emitted with
+ * payload.confirmed=undefined, then, after confirmation blocks, either with confirmed=true if tx
+ * is still present on blockchain, or confirmed=false if it got removed by a reorg.
+ *
+ * These actions must comply with the following type:
+ * {
+ *   payload: {
+ *     txHash: Hash;
+ *     txBlock: number;
+ *     confirmed: undefined | boolean;
+ *   };
+ *   meta: any;
+ * }
+ */
+export const ConfirmableActions = [
+  ChannelsActions.channelOpen.success,
+  ChannelsActions.channelDeposit.success,
+  ChannelsActions.channelWithdrawn,
```

Comment:
So close/settle aren't confirmable or just not yet implemented as comfirmable actions?

Reply:
Well the PR description says ChannelDeposit and channelWithdraw and channelOpen was done in the previous PR. So the close, settle would probably go in a new PR.

Reply:
So that means all our actions are confirmable? (That was really my question, I just danced around it)

Reply:
Most of the onchain actions that deal with balances I think should be confirmable. I remember discussing that, for example, the on-chain secret registration would not be confirmable as it would be better to know as fast as possible. But with the balances, we need to be sure that the balance changes are finalized.

Reply:
Makes sense, thank you 🙏

Reply:
All on-chain events should be confirmable. Then each epic or reducer can chose to react on pending or confirmed event only. I'm going to implement `close` & `settle` on the next PR, just splitting as we've talked about =)

Reply:
Yes I know, it was more of a general question and not specifically related to close/settle. Thank you 🙏